backend/src/navi.py

```diff
@@ -175,7 +175,7 @@ def match(
     return {"type": "match", "of": of, "arms": arms}
 
 
-def Image(
+def image(
```

Comment:
I don't like this. I think functions for specific navi types should mirror the type name. Could we ignore the lint for `image` and `color`?